manifests/b/BepassOrg/OblivionDesktop/0.35.1-beta/BepassOrg.OblivionDesktop.locale.en-US.yaml

```
@@ -0,0 +1,81 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: BepassOrg.OblivionDesktop
+PackageVersion: 0.35.1-beta
+PackageLocale: en-US
+Publisher: BepassOrg

```

```suggestion
Publisher: ircfspace+kiomarzsss
```
This should match what is written to control panel